jstests/core/distinct_multikey_index.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+    "use strict";
+
+    load("jstests/libs/analyze_plan.js");
+
+    var t = db.distinct_multikey_index;
```

Comment:
Nitpick: Rename "t" to "coll". I think "t" stands for "table", but MongoDB calls them "collections" :)